app/views/api_docs/openapi/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>API Documentation</title>
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+</head>
+<body>
+  <div id="redoc-container"></div>
+
+  <script src="https://cdn.redoc.ly/redoc/v2.1.4/bundles/redoc.standalone.js" integrity="sha384-18dIkv/vYLN5DYqrdbQr0J0VPcs5k2KUkSxAK31PYKsy+l0a9chct7ZwLDiNaBET" crossorigin="anonymous"></script>
```

Comment:
Did you try getting redoc in the app so we can avoid a remote call?

Reply:
It's now loading locally, although it wasn't straightforward! I wasn't able to use the standalone build, so had to include react for that single page. I've created separated builds so we're burdening the main application JS with the redoc JS.